desktop/main.js

```diff
@@ -374,6 +392,41 @@ const mainWindow = (() => {
                 app.hide();
             }
 
+            ipcMain.on(ELECTRON_EVENTS.LOCALE_UPDATED, (event, updatedLocale) => {
+                if (preferredLocale === updatedLocale) {
```

Comment:
Because of another suggestion in `BaseLocaleListener` that makes it so that the event isn't sent if the locale doesn't change, we can remove this check.